google/cloud/spanner/doc/spanner-main.dox

```diff
@@ -25,9 +25,9 @@ into your project.
 - [Client::Read()](@ref google::cloud::spanner::Client::Read()) - to read the
   rows in a table.
 - @ref spanner-error-handling to learn how the library reports run-time errors.
-- @ref spanner-env-logging enable logging. THis can be an effective approach to
-  diagnose many problems.
 - @ref spanner-env for environment variables affecting the library.
+- @ref spanner-env includes environment variables to enable logging. This can be
```

Comment:
s/spanner-env/spanner-env-logging/?  Although that seemed to lead nowhere.

Perhaps it is just that the two `spanner-env` entries should be combined?

Reply:
`spanner-env-logging` is not working with the DocFX-based docs, but works with Doxygen.  Merged the two entries.